### PR TITLE
Wallet SDK UI update

### PR DESCRIPTION
**Pull request by cb-jake**
### _How did you test your changes?_

Manually

Screens

Light mode
image

image

Dark mode

image

image

### CODE REVIEW

**Review by vishnumad — DISMISSED** [review]

**Review by vishnumad — APPROVED** [review]

File: packages/wallet-sdk/src/components/ConnectDialog/ConnectDialog.tsx [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+        window.setTimeout(() => {
+          setContainerHidden(true);
+        }, 360),
+      );
+    }
+
+    return () => {
+      timers.forEach(window.clearTimeout);
+    };
+  }, [props.isOpen]);
```
Comment (jeongmincho) [inline packages/wallet-sdk/src/components/ConnectDialog/ConnectDialog.tsx:51]: just curious - what are these timers for again?

**Review by jeongmincho — DISMISSED** [review]
lgtm - thanks for pushing through this gnarly UI system 🙏🏻

File: packages/wallet-sdk/src/components/Snackbar/Snackbar.tsx [hunk 2]
```diff
@@ -7,9 +7,18 @@ import { useEffect, useState } from "preact/hooks";
...
+    case "coinbase-wallet":
```
[the remark this answers is not in the document]
Comment (cb-jake) [reply packages/wallet-sdk/src/components/Snackbar/Snackbar.tsx:16]: Ahh yeah good callout I'll update the keys in `wallets`

File: packages/wallet-sdk/src/components/ConnectDialog/ConnectDialog.tsx (hunk 1 above)
Comment (cb-jake) [reply packages/wallet-sdk/src/components/ConnectDialog/ConnectDialog.tsx:51]: They're used to control the styles of the modal. They were copied over from the previous dialog

**Review by vishnumad — APPROVED** [review]

**Review by xboehmx33 — COMMENTED** [review]
lo

**Review by vishnumad — DISMISSED** [review]

**Review by jeongmincho — APPROVED** [review]

**Review by vishnumad — APPROVED** [review]

**Pull request merged**